web/views/projects.py

```diff
@@ -42,6 +43,8 @@ def project_list(request):
         'reset': True,
         'facets': facet_view_utils.filter_empty_facets(projects.facet_counts()),
         'query': query or '',
+        'filters': request.GET.get('filters') or '',
```

Comment:
maybe it should be `request.GET.get('filters', '')`?

Reply:
nice!
Refactored